docs/src/how-to-use.md

```diff
@@ -354,11 +369,58 @@ For the model to add constraints for a [maximum or minimum energy limit](@ref in
+| Defined | None              | The profile definition and value will be in the [`assets-timeframe-profiles.csv`](@ref schemas) and [`profiles-timeframe.csv`](@ref schemas) files. For example, we define a profile that has the following first four values: 0.6 p.u., 1.0 p.u., 0.8 p.u., and 0.4 p.u. There are no period partitions, so constraints will be for each period (i.e., daily). Therefore the outgoing energy of the asset for the first four days must be less than or equal to 6MWh, 10MWh, 8MWh, and 4MWh.                                                                                                                                           |
 | Defined | Defined           | Using the same profile as above, we now define a period partition in the [`assets-timeframe-partitions.csv`](@ref schemas) file as `uniform` with a value of 2. This value means that we will aggregate every two periods (i.e., every two days). So, instead of having 365 constraints, we will have 183 constraints (182 every two days and one last constraint of 1 day). Then the profile is aggregated with the sum of the values inside the periods within the partition. Thus, the outgoing energy of the asset for the first two partitions (i.e., every two days) must be less than or equal to 16MWh and 12MWh, respectively. |
 
-<!-- prettier-ignore-end -->
+## [Defining a group of assets](@id group-setup)
+
+A group of assets refers to a set of assets that share certain constraints. For example, the investments of a group of assets may be capped at a maximum value, which represents the potential of a specific area that is restricted in terms of the maximum allowable MW due to limitations on building licenses.
+
+In order to define the groups in the model, the following steps are necessary:
+
+1. Create a group in the [`groups-data.csv`](@ref schemas) file by defining the `name` property and its parameters.
+2. In the file [`assets-data.csv`](@ref schemas), assign to the assets that belong to the group the `name` of the group they belong in the `group` parameter/column.
+
+    > **Note:**
+    > A missing value in the parameter `group` in the [`assets-data.csv`](@ref schemas) means that the asset does not belong to any group.
+
+Groups are useful to represent several common constraints, the following group constraints are the ones available in the model.
+
+### [Setting up a maximum or minimum investment limit for a group](@id investment-group-setup)
+
+The mathematical formulation of the maximum and minimum investment limit for group constraints is available [here](@ref investment-group-constraints). The parameters to set up these constraints in the model are in the [`groups-data.csv`](@ref schemas) file.
+
+-   `invest_method = true`. This parameter enables the model to use the investment group constraints.
```

Comment:
Can we change it to "invest_limit" or "limit_investment" instead of method?
Method to me means the way of investing or something.
Or "constrain_investment" maybe?

Reply:
mmm...the method is Juha's style that we are trying to mimic...I don't have a strong opinion on that...Maybe we can ask @g-moralesespana what he prefers as name for the parameter...

Reply:
@clizbe I have created issue #747 to add the possibility of an asset belonging to more than one group and review the naming of the constraints' activation. So, we can leave it like this now and discuss it later when tackling the issue. Is that ok?